README.md

```diff
@@ -58,3 +58,80 @@ $ NODE_OPTIONS=--inspect-brk bin/run hello:org -u myOrg@example.com
 6. Hit the green play button at the top middle of VS Code (this play button will be to the right of the play button that you clicked in step #5).
    <br><img src=".images/vscodeScreenshot.png" width="480" height="278"><br>
    Congrats, you are debugging!
+   
+   
+## About the Predeploy Hook Example
```

Comment:
Can we have this at the top?

Reply:
I renamed my new section "About the Predeploy Hook TypeScript Code" (which better describes what the section is really about) and moved it up.